pkg/storaged/devices.jsx

```diff
@@ -88,7 +88,7 @@ class StoragePage extends React.Component {
             // we can throw this hack out.
             <>
                 <MultipathAlert client={client} />
-                <div hidden={!!detail}><Overview client={client} /></div>
+                <Overview client={client} hidden={!!detail} />
```

Comment:
This looks wrong, hidden is not a property of the Overview component. I will test it tomorrow in practice.

Reply:
It's wrong to have an empty div wrapping the page. This is why it wasn't stretching properly. 

Hidden is a property of HTML elements, and it worked properly in testing.

Reply:
Having that div in the way was causing https://github.com/cockpit-project/cockpit/issues/14534

Reply:
I just tried it out and indeed this breaks navigation to the sub pages.

I don't say that the div should be there, just the proper fix is:
```
{!detail && <Overview client={client} />}
```

Overview is a React component, that what you pass there are component properties, not html attributes.

The tests pass because your change makes both pages main and details both appear in the html.

Reply:
In addition there are in the CSS the height: 100% rules which were trying to address the same issue, and these can be removed now.